src/components/FlowRunMenu.vue

```diff
@@ -1,8 +1,7 @@
 <template>
   <p-icon-button-menu>
     <template #default>
-      <!-- todo: audit [cchoy] this PR(https://github.com/PrefectHQ/prefect-ui-library/pull/1743) will only partially work once object acls are on -->
-      <p-overflow-menu-item v-if="flowRun?.deploymentId && can.run.deployment" label="Copy to new run" :to="routes.deploymentFlowRunCreate(flowRun.deploymentId, flowRun.parameters)" />
+      <p-overflow-menu-item v-if="flowRun?.deploymentId && deployment?.can.run" label="Copy to new run" :to="routes.deploymentFlowRunCreate(flowRun.deploymentId, flowRun.parameters)" />
```

Comment:
nit: I don't think we need the `flowRun?.deploymentId` check here. `deployment?.can.run` will be undefined if there is no deploymentId right?

Reply:
uhhh. yeah I think so.

Reply:
oh actually the explicit check works as a type guard for the other half of this line where it expects a non-null `flowRun.deploymentId`